prime-dotnet-webapi/Migrations/ApiDbContextModelSnapshot.cs

```diff
@@ -260,7 +260,7 @@ protected override void BuildModel(ModelBuilder modelBuilder)
th all your duties under that Act.\n    </p>\n\n    <p>\n      The Province may, in writing and from time to time, set further limits and conditions in respect of PharmaNet,\n      either for you or for the Practitioner(s), and that you must comply with any such further limits and conditions.\n    </p>\n\n  </li>\n  <li>\n\n    <p class=\"bold underline\">\n      How to Notify the Province\n    </p>\n\n    <p>\n      Notice to the Province may be sent in writing to:\n    </p>\n\n    <address>\n      Director, Information and PharmaNet Development<br>\n      Ministry of Health<br>\n      PO Box 9652, STN PROV GOVT<br>\n      Victoria, BC V8W 9P4<br>\n\n      <br>\n\n      <a href=\"mailto:PRIMESupport@gov.bc.ca\">PRIMESupport@gov.bc.ca</a>\n    </address>\n\n  </li>\n  <li>\n\n    <p class=\"bold underline\">\n      Province may modify these terms\n    </p>\n\n    <p>\n      The Province may amend these terms, including this section, at any time in its sole discretion:\n    </p>\n\n    <ol type=\"i\">\n      <li>\n        by written notice to you, in which case the amendment will become effective upon the later of (A) the date\n        notice of the amendment is first delivered to you, or (B) the effective date of the amendment specified by the\n        Province, if any; or\n      </li>\n      <li>\n        by publishing notice of any such amendment in the PharmaCare Newsletter, in which case the notice will specify\n        the effective date of the amendment, which date will be at least thirty (30) days after the date that the\n        PharmaCare Newsletter containing the notice is first published.\n      </li>\n    </ol>\n\n    <p>\n      If you do not agree with any amendment for which notice has been provided by the Province in accordance with (i)\n      or (ii) above, you must promptly (and in any event before the effective date) cease all access or use of\n      PharmaNet.\n    </p>\n\n    <p>\n      Any written notice to you under (i) above will be in writing and delivered by the Province to you using any of the\n      contact mechanisms identified by you in PRIME, including by mail to a specified postal address, email to a\n      specified email address or text message to a specified cell phone number. You may be required to click a URL link\n      or log into PRIME to receive the contents of any such notice.\n    </p>\n\n  </li>\n  {$lcPlaceholder}\n  <li>\n\n    <p class=\"bold underline\">\n      Governing Law\n    </p>\n\n    <p>\n      These terms will be governed by and will be construed and interpreted in accordance with the laws of British\n      Columbia and the laws of Canada applicable therein.\n    </p>\n\n    <p>\n      Unless otherwise specified, a reference to a statute or regulation by name means the statute or regulation of\n      British Columbia of that name, as amended or replaced from time to time, and includes any enactment made under the\n      authority of that statute or regulation.\n    </p>\n\n  </li>\n</ol>\n",
```

Comment:
If you notice, this is replacing "\r\n" with "\n"
This text is populated from the HTML files in Configuration/Database/Resources.  I am confident you will find that these files have the "wrong" line endings on your local based on your git `autocrlf` setting. Normally, when you commit a file back to the repo it will automatically convert the line ending back. But because the file is being serialized here, your local line endings are being copied into the snapshot.

Reply:
You should check in with Nathan, he had the same issue.